frontend/js/schema-conversion.js

```diff
@@ -289,31 +290,30 @@ const createEditDataTypeTable = () => {
       if (j === 0 && globalDataTypeList[Object.keys(globalDataTypeList)[i]] !== null) {
         tableContent += `<td class='src-td' id='dataTypeKey${(i + 1)}'>${Object.keys(globalDataTypeList)[i]}</td>`;
       }
-      else if (j === 1) {
+      else if (j === 1 && globalDataTypeList[Object.keys(globalDataTypeList)[i]] !== null) {
         tableContent += `<td id='dataTypeVal${(i + 1)}'>`
```

Comment:
```
globalDataTypeList[Object.keys(globalDataTypeList)[i]] !== null
```
Above condition can be put in separate parent if condition to remove repetition of same code

Reply:
1. changes are done accordingly